h2o-algos/src/main/java/hex/tree/Score.java

```diff
@@ -40,7 +56,7 @@
     // If this is a score-on-train AND DRF, then oobColIdx makes sense,
     // otherwise this field is unused.
     final int oobColIdx = _bldr.idx_oobt();
-    _mb = m.makeMetricBuilder(domain);
+    _mb = (ModelMetricsSupervised.MetricBuilderSupervised) m.makeMetricBuilder(domain);
```

Comment:
Would it better to parameterize makeMetricBuilder by `<T>`? At least the cast would not be necessary.

Reply:
Yes, this is a good point.